dataprofiler/profilers/profile_builder.py

```diff
@@ -1420,26 +1424,32 @@ def _merge_correlation(self, other):
         if corr_mat1 is None or corr_mat2 is None:
             return None
 
-        if len(corr_mat1.columns) != len(corr_mat2.columns) or \
-                len(corr_mat1.columns) <= 1:
+        col_ids1 = np.where(~np.isnan(corr_mat1).all(axis=0))[0]
+        col_ids2 = np.where(~np.isnan(corr_mat2).all(axis=0))[0]
```

Comment:
Can you leave a comment as this is a few operations nested together.

Reply:
added